packages/headless/src/utils/compare-utils.ts

```diff
@@ -0,0 +1,27 @@
+import {createCustomEqual} from 'fast-equals';
+
+export function arrayEqualSorted<T>(firstArray: T[], secondArray: T[]) {
```

Comment:
Why is `sorted` in the name? The code is only checking that the values are the same at every index. It is not sorting the elements?

If two arrays are equal, then it typically means the order is the same, so why the extra word?